components/omnibox/browser/brave_omnibox_prefs.cc

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/omnibox/browser/brave_omnibox_prefs.h"
+
+#include "components/prefs/pref_registry_simple.h"
+
+namespace omnibox {
+
+const char kAutocompleteEnabled[] = "brave.autocomplete_enabled";
+const char kTopSiteSuggestionsEnabled[] = "brave.top_site_suggestions_enabled";
+const char kBraveSuggestedSiteSuggestionsEnabled[] =
+    "brave.brave_suggested_site_suggestions_enabled";
+const char kHistorySuggestionsEnabled[] =
```

Comment:
can you please add comments to these prefs? especially this one pref, which covers a lot of providers, should mention that.